scholia/app/templates/author_curation.html

```diff
@@ -1,4 +1,4 @@
-{% extends "base.html" %}
+{% extends "q_curation.html" %}
```

Comment:
Could it be that you forgot to add the q_curation.html template to the commit?

Reply:
yep